### PR TITLE
updateding the linters

### PR DESCRIPTION
**Pull request by PowerLevel9000**
Reverts PowerLevel9000/hello-microverse#2

### CODE REVIEW

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @PowerLevel9000 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* Meaningful commit messages ✔️ 
* Github flow is followed correctly ✔️ 

## Required Changes:
- [ ] Kindly add an `index.html` file and `style.css` file as mentioned in the requirements.
- [ ] Kindly update the description of the PR to make it descriptive of the changes made in the branch 👍 
- [ ] From now on, kindly choose the branch's name carefully it should be descriptive of the actions made in the branch 👍 
- [ ] Kindly update the title of the PR to make it descriptive of what's going on in the PR. ex: "setup hello microverse"...
- [ ] Kindly put the `linters.yml` file inside `workflows` directory to have linters checks running correctly as showed in this [guide](https://github.com/microverseinc/linters-config/tree/master/html-css) 💯 
- [ ] Kindly update your README file, you should follow this [template](https://github.com/microverseinc/readme-template), copy all of the content and fill out the necessary sections with information of your project to have a professional documentation ✨ 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PowerLevel9000 — COMMENTED** [review]
i have made changes there 
    - added pr 
    - linter in this branch 
    - index and css file in branch
    kindly review my code

File: README.md [hunk 1]
```diff
@@ -6,43 +6,36 @@
 -HTML
 -CSS</a>
```
Comment (Kamba56) [inline README.md:8]: - [ ] As requested by the previous reviewer, kindly correct your linters setup. Note that the `linters.yml` file should be inside `.github/workflows` folder. The `workflows` is a folder inside `.github` then move the `linters.yml` into the workflows folder.

- [ ] As requested by the previous reviewer, kindly edit your pull request heading and description. The pull request heading should be a description of what was implemented on the branch while the description should be a summary of all the changes implemented. Check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information.

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @PowerLevel9000 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**